src/appleseed/renderer/kernel/shading/closures.cpp

```diff
@@ -1241,6 +1241,7 @@ namespace
             OSL::Color3     mean_free_path;
             float           ior;
             float           fresnel_weight;
+            float           zero_scattering_weight;
```

Comment:
There are a few missing bits:

You need to initialize zero_scattering_weight with a default value here:
https://github.com/appleseedhq/appleseed/pull/1842/commits/08bb52777280d2dda2608b068704f99b1af62435#diff-84fe0468350bc438ced5b170a9197da1L1263

In case the user did not specify it as an argument.

And you need to copy the value to the BSSDF inputs here:

https://github.com/appleseedhq/appleseed/pull/1842/commits/08bb52777280d2dda2608b068704f99b1af62435#diff-84fe0468350bc438ced5b170a9197da1L1384

Reply:
But before that, we should first check with Artem if this parameter will still exist in his improved RandomWalkBSSRDF.

Reply:
Artem told me that he is going to remove the zero scattering weight param from the bssrdf.
We could just remove the changes in this file.

Reply:
@est77 i'll remove this from as_subsurface as well.